wplib.php

```diff
@@ -375,6 +508,20 @@ private static function _find_autoload_files() {
 	}
 
 	/**
+	 * Force loading of all classes if needed to find all classes with a specific constant.
+	 */
+	static function autoload_all_classes() {
```

Comment:
How often / when is this run?  Doesn't this defeat the purpose of an autoloader?